add_events.py

```diff
@@ -112,8 +112,12 @@ def main():
                 replaceHour += int(startHour)
             event = {}
             # Currently there are no labs in `subjects.json`
-            if (data[day][time][0] in subjects.keys()):
-                event['summary'] = "Class of " + subjects[data[day][time][0]].title()
+            # if (data[day][time][0] in subjects.keys()):
+            if (data[day][time][2] < 3):
```

Comment:
@nishnik The lab hours are decided using the `L-T-P` rating of a course. So, a 0-0-2 course is a 2 hour lab course.